tests/zpill.py

```diff
@@ -240,8 +241,10 @@ def assertJmes(self, expr, instance, expected):
     def cleanUp(self):
         pass
 
-    def record_flight_data(self, test_case, zdata=False):
+    def record_flight_data(self, test_case, zdata=False, vary_on_runtime=False):
```

Comment:
this feels a little odd tbh, both on the helper addition, and on the caller, i'd rather the tests be deterministic, ie. if we want to test both runtimes, then let's test both, afaics the caller doesn't even care per se. updating the tests to just match default_runtime in their asserts would avoid the helper.

Reply:
I've replaced 71c8b6bf7bf1c6033c13162e3fb8e8bd2f2e2b33 with 9855d4dd8b6c5e3a883e5ba4f554ef54a52892ea, which uses `python2.7` as the default runtime for policy lambdas regardless of the local runtime. Better?